src/rc/start-stop-daemon.c

```diff
@@ -372,6 +381,28 @@ int main(int argc, char **argv)
 #endif
 			break;
 
+		case 0x101:
+#ifdef HAVE_CAP
+			if (*optarg == '\0')
+				eerrorx("Secbits are empty");
+
+			tmp = NULL;
+			secbits = strtoul(optarg, &tmp, 0);
+			if (*tmp != '\0')
+				eerrorx("Could not parse secbits: invalid char %c", *tmp);
+#else
+			eerrorx("Capabilities support not enabled");
+#endif
+			break;
+
+		case 0x102:
+#ifdef __linux__
```

Comment:
let's use `PR_SET_NO_NEW_PRIVS` instead of `__linux__`.  it's "new" to linux-3.5.